storage/reads/array_cursor.gen.go.tmpl

```diff
@@ -244,15 +317,15 @@ func (c {{$type}}) Next() {{$arrayType}} {
 
 {{end}}
 
-type integer{{.Name}}CountArrayCursor struct {
+type {{.Name}}CountArrayCursor struct {
```

Comment:
This was actually a change to make this match the original code a little more. the `interger` before these types is redundant.